src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -124,17 +127,22 @@ class ReportActionContextMenu extends React.Component {
                     );
                 },
             },
-
             {
                 text: this.props.translate('reportActionContextMenu.deleteComment'),
                 icon: Trashcan,
-                shouldShow: false,
-                onPress: () => {},
+                shouldShow: this.isUserActorOfAction(),
```

Comment:
You probably also want to include `&& this.props.reportAction.reportActionID,` here - since you would not be able to delete a comment unless you had the reportActionID (and that only comes once a comment has been written to the DB via the API)

Reply:
is this the same case for the edit?

Reply:
done

Reply:
Yeah same case for edit